src/main/java/mike/cli/CommandLine.java

```diff
@@ -12,10 +10,11 @@
 	public CommandLine() {
 	}
 
-	public static void commandInterface() throws IOException {
-		Classes userClasses = new Classes();
+	public static void commandInterface() {
+		datastructures.Classes userClasses = new Classes();
```

Comment:
Maybe this was to get Eclipse to work? But shouldn't need datastructures. in front of Classes since you import it. Throws an error on my machine when using Gradle.

Reply:
Fixed. Don't even know when that got changed.